cabin-sorting/api/sortingData.ts

```diff
@@ -0,0 +1,70 @@
+import path from "path";
+import fs from "fs";
+import { client, connectToDatabase } from "./connectMongo";
+import { Hacker } from "../hackerSortingAlgo";
+
+let validatedHackers : Hacker[] = [];
+const validHackers : Hacker[] = [];
+
+// Get all valid hacker data from the applicant_data database and convert to json contents
+async function grabFromDatabase() {
+  const applicantData = await connectToDatabase("applicant_data");
+
+  try {
+    const query = {
+      applicationStatus: "Submitted",
+      postAcceptanceResponses: { $exists: true },
+      isAdmin: false,
+      rsvpStatus: "Confirmed",
+      "postAcceptanceResponses.swag": { $exists: false },
+      "postAcceptanceResponses.club": { $exists: true },
+    };
+
+    const hackerDataCursor = applicantData
+      .find(query)
+      .project({ email: 1, postAcceptanceResponses: 1 });
+
+    
+    const hackerDocuments = await hackerDataCursor.toArray();
+    hackerDocuments.map((item) => parseAndFormatHacker(item));
+    validatedHackers = validateHackers(validHackers);
+
+    const hackerJson = JSON.stringify(validatedHackers);
+    const pathToWrite = path.resolve(
+      __dirname,
+      "../",
+      "data",
+      "json_outputs",
+      "fromMongoDB.json"
+    );
+    fs.writeFileSync(pathToWrite, hackerJson);
+  } finally {
+    await client.close();
+  }
+  return validatedHackers;
+}
+
+// used for parsing hacker data into a uniform, correct, format
```

Comment:
this comment does not add value. either delete or if needed, elaborate on what "uniform / correct format" means

Reply:
also, renaming function can help clarify